src/test/java/com/exasol/adapter/jdbc/BaseColumnMetadataReaderTest.java

```diff
@@ -46,15 +44,31 @@ void testMappingUnsupportedTypesReturnsUnsupportedType(final int jdbcType) {
     void testGetColumnsFromResultSetSkipsUnsupportedColumns() throws SQLException {
         final ResultSet remoteColumnsMock = mock(ResultSet.class);
         when(remoteColumnsMock.next()).thenReturn(true, true, true, false);
-        when(remoteColumnsMock.getString(BaseColumnMetadataReader.NAME_COLUMN)).thenReturn("DATE_COL", "BLOB_COL",
-                "DOUBLE_COL");
-        when(remoteColumnsMock.getInt(BaseColumnMetadataReader.DATA_TYPE_COLUMN)).thenReturn(Types.DATE, Types.BLOB,
-                Types.DOUBLE);
+        when(remoteColumnsMock.getString(BaseColumnMetadataReader.NAME_COLUMN))
+              .thenReturn("DATE_COL", "BLOB_COL", "DOUBLE_COL");
+        when(remoteColumnsMock.getInt(BaseColumnMetadataReader.DATA_TYPE_COLUMN))
+              .thenReturn(Types.DATE, Types.BLOB, Types.DOUBLE);
         final List<ColumnMetadata> columns = this.reader.getColumnsFromResultSet(remoteColumnsMock);
         final List<ExaDataType> columnTypes = columns //
-                .stream() //
-                .map(column -> column.getType().getExaDataType()) //
-                .collect(Collectors.toList());
+              .stream() //
+              .map(column -> column.getType().getExaDataType()) //
+              .collect(Collectors.toList());
         assertThat(columnTypes, containsInAnyOrder(ExaDataType.DATE, ExaDataType.DOUBLE));
     }
+
```

Comment:
This code looks copied from another module, right?
Did you remove the duplication?

Reply:
I don't remember if I copied it or not.    ლ(ಠ_ಠ ლ)
And I can't find the duplication.